Dockerfile

```diff
@@ -50,6 +50,10 @@ RUN dpkg --add-architecture i386 && \
         libstdc++6:i386 \
         libsdl1.2-dev
 
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN apt clean
```

Comment:
According to the documentation, official Debian/Ubuntu images are already this run
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

so I think this is not needed

Reply:
okay